runconfig/opts/parse.go

```diff
@@ -59,7 +59,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 		flPrivileged        = cmd.Bool([]string{}, false, "Give extended privileges to this container")
 		flPidMode           = cmd.String([]string{}, "", "PID namespace to use")
 		flUTSMode           = cmd.String([]string{}, "", "UTS namespace to use")
-		flPublishAll        = cmd.Bool([]string{}, true, "Publish all exposed ports to random ports")
+		flPublishAll        = cmd.Bool([]string{"P", "-publish-all"}, true, "Publish all exposed ports to random ports")
```

Comment:
The server keep the `PublishPorts` be true, I want to make sure both client and server are consistent.